### ISSUE TITLE
Toxic Exposure Accessibility Updates for MVP

### ISSUE THREAD

**Issue by Kate-allbee** [department-of-veterans-affairs/va.gov-team#72701]
## Issue Description
Implement accessibility feedback given from CAIA

---
## Tasks

- [x] Update colors listed in CAIA doc to pass accessibility
- [x] Update date components to meet design system standards
- [x] Use accessibility annotations before handoff to dev (Label headings, ARIA text) > migrate labels to Figma
- [x] Use VADS color palette for buttons, progress bar, checkboxes
- [x] Use VADS text area component for better usability
- [x] Revise page headers to be more specific than "Toxic exposure"

## Acceptance Criteria

- [x] All colors are to accessibility standards
- [x] All components match what is in the design system
- [x] Accessibility annotations for handoff to dev

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by mengA6** [comment]
Completed in both Sketch file and [Figma file](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=0%3A411&mode=design&t=Jhq380TFg6mdiamn-1) (Figma is the current source of truth)

**Comment by mengA6** [comment]
We might get some accessibility guidance from [my question on how to structure the HTML for the summary screen](https://dsva.slack.com/archives/C8E985R32/p1704903482413669), but we might not.

**Comment by mengA6** [comment]
Updated [figma file](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=0%3A411&mode=design&t=Jhq380TFg6mdiamn-1) with new guidance on how to structure summary screen content

**Comment by mengA6** [comment]
Updated [figma file](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=0%3A411&mode=design&t=Jhq380TFg6mdiamn-1) to use headings but style them to appear with bullets for ease of visual scanning

**Comment by mengA6** [comment]
Updated [figma file](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=0%3A411&mode=design&t=Jhq380TFg6mdiamn-1) with [more detailed guidance from CAIA](https://dsva.slack.com/archives/C8E985R32/p1705154213264839?thread_ts=1704903482.413669&cid=C8E985R32) on how to structure the HTML.
